deployment/terraform/engine.go

```diff
@@ -118,3 +118,14 @@ func checkTerraformVersion() error {
 
 	return nil
 }
+
+// checkAWSCLI checks that the aws command is available in the system, and that
+// the profile that will be used is correctly configured
+func checkAWSCLI(profile string) error {
+	cmd := exec.Command("aws", "configure", "list", "--profile", profile)
+	err := cmd.Run()
```

Comment:
Just a nitpick: You can write those two as one line, since you're not using `cmd` anywhere else.

Reply:
That's true, but I think this way is a bit more readable. I'll take the liberty to keep it since it's a nitpick :)